php/libraries/Candidate.class.inc

```diff
@@ -184,7 +185,7 @@ class Candidate
      *
      * @param int         $centerID              centerID of the site to which the
      *                                           candidate will belong
-     * @param string      $dateOfBirth           date of birth written as YYYY-MM-DD
+     * @param string|null $dateOfBirth           date of birth written as YYYY-MM-DD
```

Comment:
@maltheism when is the dob ever null ?

Reply:
its a required field for participants

Reply:
@ridz1208 we had a loris discussion about this PR. dob can be null from what I remember was said at the loris meeting but I don't remember the exact example. I guess when the subject is expected to be born.

Reply:
as far as I can tell in 23 DOB is still a required field so I don't see when this case occurs so this seems like it's allowing it where it shouldnt

Reply:
@ridz1208 I'll bring it up at next loris meeting.

Reply:
looking at the other PR i think that's the usecase for making it nullable

`3. birthday should not be required if using edc.`

it's fine then especially if it was discussed in the LORIS meeting

Reply:
@maltheism could you update the description to include the change so it doesn't seem accidental

Reply:
@ridz1208 the comment description?

Reply:
no the description of the PR itself on top